.github/workflows/catnip.yml

```diff
@@ -46,6 +46,28 @@ jobs:
         echo -e "\tPasswordAuthentication no" >> $HOME/.ssh/config
         echo -e "\tUser ${{ secrets.USERNAME }}" >> $HOME/.ssh/config
         echo -e "\tPort ${{ secrets.PORTNUM }}" >> $HOME/.ssh/config
+    - name: Bootstrap VMs
+      shell: bash
+      run: |
+        echo "Running azure.sh on SERVER"
+        ssh $SERVER "sudo bash ~/demikernel/scripts/setup/azure.sh"
+
+        echo "Running hugepages.sh on SERVER"
+        ssh $SERVER "sudo bash ~/demikernel/scripts/setup/hugepages.sh"
+
+        echo "Running ip addr flush on SERVER"
+        ssh $SERVER "sudo ip -br addr show dev eth1 | grep -q ' UP ' && \
+                     sudo ip addr flush dev eth1"
```

Comment:
It is not guaranteed (nor safe) to assume that  `eth1` is the network interface that we should target (this may change on reboot). A safer automated proceedure would query the target interface based on the IP address (which we can get from our secrets).

Reply:
Can you share how this is done in the startup scripts? I will reuse the same mechanism here as well.

Reply:
We discussed offline, and will fix it if (and when) required.